### ISSUE TITLE
update Quarkus version to 2.6 or later (I'd prefer 2.8.1 personally)

### ISSUE THREAD

**Issue by SingingBush** [Nithanim/quarkus-dynamodb-enhanced#5]
Please update the project to use the latest Quarkus version and fix the dependencies. In 2.6 Quarkus had a bit of a reshuffle and moved some dependencies which result in warning like this in my build:

```
The artifact io.quarkus:quarkus-amazon-dynamodb-deployment:jar:2.8.1.Final has been relocated to io.quarkiverse.amazonservices:quarkus-amazon-dynamodb-deployment:jar:1.0.2: io.quarkus:quarkus-amazon-dynamodb-deployment was relocated to io.quarkiverse.amazonservices:quarkus-amazon-dynamodb-deployment and is now managed by the io.quarkus.platform:quarkus-amazon-services-bom:2.8.1.Final. Please, update its groupId in the dependency declaration and import io.quarkus.platform:quarkus-amazon-services-bom:2.8.1.Final in your project configuration. For more information about this change, please, refer to https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.6
```

For more details read https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.6

Basically change:

```xml
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-amazon-dynamodb</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-amazon-dynamodb-deployment</artifactId>
        </dependency>
```

to:

```xml
        <dependency>
            <groupId>io.quarkiverse.amazonservices</groupId>
            <artifactId>quarkus-amazon-dynamodb</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkiverse.amazonservices</groupId>
            <artifactId>quarkus-amazon-dynamodb-deployment</artifactId>
        </dependency>
```

**Comment by Nithanim** [comment]
Thank you very much for reminding me! I will look into it this weekend.

**Comment by Nithanim** [comment]
I just published the new release of version `1.1.0` to maven central which is built against Quarkus 2.8.2. Please have some patience while it propagates to the mirrors. You can check https://repo1.maven.org/maven2/me/nithanim/quarkus/quarkus-amazon-dynamodb-enhanced/ directly if you want.

Please do not hesitate to contact me if you encounter any problems!
